integration_tests/src/test/kotlin/integration/SanityRoboIT.kt

```diff
@@ -0,0 +1,30 @@
+package integration
+
+import FlankCommand
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import run
+
+class SanityRoboIT {
+    private val name = this::class.java.simpleName
```

Comment:
and again ;)

Reply:
https://github.com/Flank/flank/pull/1316#discussion_r522089515